19/transaction_validator_election_vote_v2.0.yaml

```diff
@@ -0,0 +1,31 @@
+---
+"$schema": "http://json-schema.org/draft-04/schema#"
+type: object
+title: Validator Election Vote Schema - Vote on a validator set change
+required:
+- operation
+- version
+- outputs
```

Comment:
What do outputs of the vote transaction describe?

Reply:
To whom the cost is being casted to. The voter can choose to delegate their vote or vote to the `election_id`.